packages/editor-ui/src/components/MfaSetupModal.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+	<Modal
+		width="500px"
+		title="Multi-factor Authentication Setup"
+		:eventBus="modalBus"
+		:name="MFA_SETUP_MODAL_KEY"
+		:center="true"
+	>
+		<template #content>
```

Comment:
We should add a "loading" state here, to prevent a QR code for an empty string being rendered temporarily. This modal should only be rendered after the call for the qr code succeeds.